frontend/src/app/navigation/navigation.component.ts

```diff
@@ -73,6 +75,14 @@ export class NavigationComponent implements OnInit, OnDestroy {
     }
   }
 
+  onSlackInviteClick(): void {
+    const dialogRef = this.slackDialog.open(SlackInviteBox, {
+      width: '1000px',
+      autoFocus: false
```

Comment:
I'm going to modify these to be more aligned with latest dialog options and remove the hardcoded width.